src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/PackageWrapper.cs

```diff
@@ -48,7 +48,7 @@ public static PackageWrapper GetWrapperFromPackage(Package package)
             string path;
             try
             {
-                path = package.InstalledLocation.Path;
+                path = package.InstalledPath;
```

Comment:
`InstalledPath` is only available starting from 19041 build.

Reply:
Because we support 18362 as well we should make it conditional.

Reply:
think at the start we should do a static one time API contract check so we're not slamming then check that bool

Reply:
Yup! Added the check as a Lazy<bool> :)